lldb/test/Shell/helper/build.py

```diff
@@ -683,14 +683,14 @@ def _get_compilation_command(self, source, obj):
             args.append("-fms-compatibility-version=19")
         args.append("/c")
 
+        if self.std:
```

Comment:
Why is this change necessary?

Reply:
`/std` was appended after the `--`, so the compiler interpreted it as a file.

Reply:
I see, lets do that in a separate patch please

Reply:
Sorry, I should've asked if it's supposed to be a separate PR or commit - opened https://github.com/llvm/llvm-project/pull/141782 now.